src/main/java/Lists.java

```diff
@@ -1,38 +1,38 @@
 import java.util.ArrayList;
 
 public class Lists {
-  private static ArrayList<Task> list = new ArrayList<Task>();
+    private static ArrayList<Task> list = new ArrayList<Task>();
 
-  public static void addItem(String item) {
-    Task t = new Task(item);
-    list.add(t);
-    Messages.line();
-    System.out.println("Added: " + item);
-    Messages.line();
-  }
+    public static void addItem(String item) {
```

Comment:
Using a more descriptive name like ``addTask`` would allow better clarity

```suggestion
    public static void addItem(String item) {
```